clang-tools-extra/test/clang-tidy/checkers/modernize/use-nullptr.cpp

```diff
@@ -84,6 +84,29 @@ void test_macro_expansion4() {
 #undef MY_NULL
 }
 
+template <typename T> struct pear {
+  // If you say __null (or NULL), we assume that T will always be a pointer
```

Comment:
why are we only testing one of those?

Reply:
That was the whole thing we were belaboring earlier -- this test defines `NULL` to `0` and thus isn't caught, but in production, it's defined as `__null`, where the new code catches it. We don't actually have code to detect "this was spelled as a macro", I'm afraid (right, @zygoloid?).

Reply:
Sorry, the implication of my comment was: explain why in a comment, because future readers will not know that.

Reply:
Sure, done.

Reply:
Thanks, otherwise this comment will only cause confusion.